samples/highcharts/accessibility/accessible-pattern-fills/demo.js

```diff
@@ -0,0 +1,396 @@
+            chartOptions: {
+                plotOptions: {
+                    series: {
+                        dataLabels: {
+                            format: '<b>{point.name}</b>'
+                        }
+                    }
+                }
+            }
+        }]
+    }
+});
+
+Highcharts.chart('container4', {
+    chart: {
+        type: 'spline',
+        style: {
+            fontFamily: 'Roboto,Arial'
+        }
+    },
+
+    exporting: {
+        enabled: false
+    },
+
+    legend: {
+        enabled: false,
+        symbolWidth: 40
+    },
+
+    title: {
+        text: ''
+    },
+
+    yAxis: {
+        visible: false,
+        title: {
+            text: 'Percentage usage'
+        }
+    },
+
+    xAxis: {
+        visible: false,
+        title: {
+            text: 'Time'
+        },
+        accessibility: {
+            description: 'Time from December 2010 to September 2019'
+        },
+        categories: ['December 2010', 'May 2012', 'January 2014', 'July 2015', 'October 2017', 'September 2019']
+    },
+
+    tooltip: {
+        valueSuffix: '%',
+        enabled: false
+    },
+
+    plotOptions: {
+        series: {
+            label: {
+                enabled: false
+            },
+            point: {
+                events: {
+                    click: function () {
+                        window.location.href = this.series.options.website;
+                    }
+                }
+            },
+            cursor: 'pointer'
+        }
+    },
+
+    series: [{
+        name: 'NVDA',
+        data: [34.8, 43.0, 51.2, 41.4, 64.9, 72.4],
+        website: 'https://www.nvaccess.org',
+        color: colors[2],
+        accessibility: {
+            description: 'This is the most used screen reader in 2019'
+        }
+    }, {
+        name: 'JAWS',
+        data: [69.6, 63.7, 63.9, 43.7, 66.0, 61.7],
+        website: 'https://www.freedomscientific.com/Products/Blindness/JAWS',
+        dashStyle: 'ShortDashDot',
+        color: colors[0]
+    }, {
+        name: 'VoiceOver',
+        data: [20.2, 30.7, 36.8, 30.9, 39.6, 47.1],
+        website: 'http://www.apple.com/accessibility/osx/voiceover',
+        dashStyle: 'ShortDot',
+        color: colors[1]
+    }, {
+        name: 'Narrator',
+        data: [null, null, null, null, 21.4, 30.3],
+        website: 'https://support.microsoft.com/en-us/help/22798/windows-10-complete-guide-to-narrator',
+        dashStyle: 'Dash',
+        color: colors[9]
+    }, {
+        name: 'ZoomText/Fusion',
+        data: [6.1, 6.8, 5.3, 27.5, 6.0, 5.5],
+        website: 'http://www.zoomtext.com/products/zoomtext-magnifierreader',
+        dashStyle: 'ShortDot',
+        color: colors[5]
+    }, {
+        name: 'Other',
+        data: [42.6, 51.5, 54.2, 45.8, 20.2, 15.4],
+        website: 'http://www.disabled-world.com/assistivedevices/computer/screen-readers.php',
+        dashStyle: 'ShortDash',
+        color: colors[3]
+    }],
+
+    responsive: {
```

Comment:
Can remove this entire responsive section, given that the axes are invisible and the legend is disabled.